src/ratelimit_manager.cpp

```diff
@@ -67,11 +68,19 @@ bool RateLimitManager::is_rate_limited(const std::vector<rate_limit_entity_t> &e
             if (throttled_entities.at(entity).throttling_to > std::chrono::system_clock::to_time_t(std::chrono::system_clock::now())) {
                 return true;
             }
+
+
+            // Remove ban from DB store
+            std::string ban_key = std::string(BANS_PREFIX) + "_" + std::to_string(throttled_entities.at(entity).status_id);
```

Comment:
The ban key construction logic must be a helper method because it's duplicated in atleast one another place.